script.js

```diff
@@ -610,14 +610,14 @@ function saveCharacter() {
     };
     
     // First save as JSON for data portability
-    const characterJSON = JSON.stringify(character, null, 2);
-    const jsonBlob = new Blob([characterJSON], { type: 'application/json' });
-    const jsonUrl = URL.createObjectURL(jsonBlob);
-    const jsonLink = document.createElement('a');
-    jsonLink.href = jsonUrl;
-    jsonLink.download = 'storypath-character.json';
-    jsonLink.click();
-    URL.revokeObjectURL(jsonUrl);
+    // const characterJSON = JSON.stringify(character, null, 2);
+    // const jsonBlob = new Blob([characterJSON], { type: 'application/json' });
+    // const jsonUrl = URL.createObjectURL(jsonBlob);
+    // const jsonLink = document.createElement('a');
+    // jsonLink.href = jsonUrl;
+    // jsonLink.download = 'storypath-character.json';
+    // jsonLink.click();
+    // URL.revokeObjectURL(jsonUrl);
```

Comment:
We use git for a reason, no need to comment, just delete it.

Reply:
This is the Green Telecom company convertional style ;)